### ISSUE TITLE
Out of memory error

### ISSUE THREAD

**Issue by Silberling** [ceefour/wincachegrind#5]
Tried to open a 8 GB PHP generated cachegrind file and after some time i got an out of memory error. I assume this is because it is a 32bit application? Is there anything I can do about it except of compiling it by myself?

**Comment by ceefour** [comment]
Yep, and I think even if you got it to work with 64-bit, it'd still be impractical/slow to use unless you heavily optimize WinCacheGrind's code.

**Comment by Quix0r** [comment]
God this error with even a 2 GB large file on a 8 GB (Win7) system.

**Comment by FFMG** [comment]
The limit on a 32bit process is more like 1.7Gb, the OS will reserve some memory for itself and the process uses some memory before the file is even loaded. 

The *total* memory cannot be more than 2Gb, that includes what is needed to start the app and so on.
So, 2Gb will not open.

**Comment by Quix0r** [comment]
I know. :-) Still he should consider providing a 64-bit binary ...

**Comment by ceefour** [comment]
Contributors welcome :)
